### PR TITLE
Made Punctuation, Grammar and other changes in the documentation - Issue 104

### PR DESCRIPTION
**Pull request by devTejas**
# Description

Removed all the grammatical, punctuation errors from the documentation and other readme files.

## Fixes #(104)

Screenshot (208)
Screenshot (209)

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Friday/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Added new sections such as `Major Issues and Updates` and `Resources`

## Checklist:

- [x] My changes follows the style guidelines of this project.
- [x] I have performed a self-review of my own changes.
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by devTejas** [comment]
Hey @avinashkranjan, I have resolved the conflicts and made a new branch.
Thank you.

**Review by hareshnayak — APPROVED** [review]
There is no change in file " CODE_OF_CONDUCT.md "
You have added only one word "the" in " CONTRIBUTING.md "
Other two file have some minor changes.
If you want to contribute to the document you could write some original content you find better for the doc.

For now I am approving the PR , but it would be appreciated if you could provide more original content

File: README.md [hunk 1]
```diff
@@ -1,59 +1,97 @@
...
+
+<h1 align=center> Project Admin ❤️ </h1>
+<p align="center">
+  <a href="https://github.com/avinashkranjan"><img src="https://user-images.githubusercontent.com/55796944/95675026-dab07580-0bd1-11eb-93e2-1cb1de8acf38.png" width=150px height=150px /></a>
+
+<p align="center">
+  <img src="https://img.shields.io/badge/avinashkranjan%20-%230077B5.svg?&style=for-the-badge&logo=linkedin&logoColor=white"/>  <img src="https://img.shields.io/badge/iavinashranjan%20-%231DA1F2.svg?&style=for-the-badge&logo=Twitter&logoColor=white"/> <img src="https://img.shields.io/badge/avinashkranjan7%20-%23E4405F.svg?&style=for-the-badge&logo=Instagram&logoColor=white"/>
+
+## Contributors ✨
+
```
Comment (avinashkranjan) [inline README.md:67]: @devTejas Kinly Make Changes in the New `README` as README.md as been updated..

**Review by avinashkranjan — CHANGES_REQUESTED** [review]

**Comment by avinashkranjan** [comment]
No Response Closing this `PR`

**Pull request closed without merging**